esp-hal/src/rmt.rs

```diff
@@ -87,7 +87,7 @@ use core::{
 };
 
 use embassy_sync::waitqueue::AtomicWaker;
-use enumset::{EnumSet, EnumSetType};
+use enumset::{EnumSet, EnumSetType, __internal::EnumSetTypeRepr};
```

Comment:
Isn't there a better API than this?

Reply:
🤔 Clippy teamed up with VsCode auto-import to fool me apparently